### DISCUSSION TITLE
How far can you go back in time to update tracker?

### DISCUSSION THREAD

**Discussion by sjboyd81** [iSoron/uhabits#1704] [Help & FAQ]
I'm trying to update a habit to the beginning of March 2023 (it's currently May 3), but it will only let me scroll back to the 6th of March. Is there a way to get further back?

**Comment by jackwbradley** [comment]
I'm wondering the exact same thing! Want to update a habit to the start of the year but can only go back 2 months.

**Comment by ezio-melotti** [comment]
You should be able to tap on the habit you want to update, scroll down to the [Calendar section](https://github.com/iSoron/uhabits/blob/dev/screenshots/5.png?raw=true) (after Target, Score, and History), and tap on edit.  From there you can select and update any day.
